packages/react-utilities/src/utils/getNativeElementProps.ts

```diff
@@ -24,7 +24,7 @@ import {
   getNativeProps,
 } from './properties';
 
-const nativeElementMap: Record<string, Record<string, number>> = {
+const nativeElementMap: Record<keyof JSX.IntrinsicElements, Set<string>> = {
```

Comment:
Looking at your errors in CI, we should probably do a `Pick` of the elements that we explicitly support here

```suggestion
const nativeElementMap: Record<keyof Pick<JSX.IntrinsicElements, 'a' | 'button>, Set<string>> = {
```

Reply:
You'll probably have to rethink the args in `getNativeProps` and `getNativeElementProps` to make it work with component usages.

Reply:
Yeah I forgot `div` and `span`, so umm...will definitely be fixing that